go.mod

```diff
@@ -85,6 +84,7 @@ require (
 	github.com/go-playground/universal-translator v0.18.0 // indirect
 	github.com/godbus/dbus v0.0.0-20190726142602-4481cbc300e2 // indirect
 	github.com/gogo/googleapis v1.4.1 // indirect
+	github.com/gogo/protobuf v1.3.2 // indirect
```

Comment:
What changes does this require from the upstream library(go-sdk, storage-provider, juno)? It looks like the dependent version of gogoprotobuf has been down to v1.3.2, is it compatible?

Reply:
right, cosmos sdk require this version of gogo proto.